File: perl/tests/500-index-bourreaux.pl
my $admin      = &AdminHandle;
our $bourreau_list_500 = $admin->index_bourreaux();

if ($DEBUG) {
  my $top_entries = &ShowTopEntries($bourreau_list_500,4);
  print "LIST (first bourreaux)=\n${top_entries}\n";
}

die "Value returned not an array.\n" unless defined($bourreau_list_500) && ref($bourreau_list_500) eq "ARRAY";

1;

File: ruby/tests/500-index-bourreaux.rb
admin              = admin_handle
@bourreau_list_500 = admin.index_bourreaux()

puts "LIST (first bourreaux)= \n#{show_top_entries(@bourreau_list_500,5)}" if @debug

raise "Value returned by index_bourreaux() is not an array" unless @bourreau_list_500.present? && @bourreau_list_500.is_a?(Array)

true
